android/app/src/main/AndroidManifest.xml

```diff
@@ -119,6 +119,13 @@
         <data android:scheme="ioit" android:host="ioit" />
       </intent-filter>
 
+      <intent-filter>
+        <action android:name="android.intent.action.VIEW" />
+        <category android:name="android.intent.category.DEFAULT" />
+        <category android:name="android.intent.category.BROWSABLE" />
+        <data android:scheme="iowalletcie" />
```

Comment:
why scheme are not equal between ios and android?

Reply:
I was unable to test on iOS. I need a bit of support to adjust these schemes.

Reply:
Given the issue you encountered I'd say that we have the following situation: 
- A schema for `io-react-native-login-utils` which is defined at package level for Android and doesn't need an entry for iOS;
- A schema for the RN linking module which requires an entry for both Android and iOS.

I think we can align them to `iowalletcie` for both of them. Does it make sense?

Reply:
It does. I've updated the scheme in the Info.plist file.